post_office/mail.py

```diff
@@ -58,14 +59,17 @@ def create(sender, recipients=None, cc=None, bcc=None, subject='', message='',
     else:
 
         if template:
+            language = template.language
             subject = template.subject
             message = template.content
             html_message = template.html_content
-
-        _context = Context(context or {})
-        subject = Template(subject).render(_context)
-        message = Template(message).render(_context)
-        html_message = Template(html_message).render(_context)
+        else:
+            language = ''
+        with translation_override(language):
+            _context = Context(context or {})
+            subject = Template(subject).render(_context)
+            message = Template(message).render(_context)
```

Comment:
`post-office` shouldn't implicitly translate the template itself. The translation should be explicitly defined by the user during the creation of the email template.

Many languages that are based on english alphabets (e.g Indonesian) often use mixed english/Indonesian content on purpose for marketing reasons.